### DISCUSSION TITLE
Use flutter/pocketbase in token collections with the SDK?

### DISCUSSION THREAD

**Discussion by orangearmandi** [pocketbase/pocketbase#5606] [Q&A]
hello, a cordial greeting and a big favor, is that I am trying to use the sdk with flutter and I want to use the token in a collection apart from the user, an example I have a collection called alerts but I do not know what rule is available to invoke the token and be able to use the token as security and thus be able to use list, create, delete ... since the example that is had, does not specify the use of the token, I am passionate about this technology, what's more my degree project depends on it, and the only thing that stops us is security: c since anyone can use the api just by having the ip.
![token](https://github.com/user-attachments/assets/d81626dd-6a70-4715-98e8-c55ea7475763)
![token 2](https://github.com/user-attachments/assets/3cfb636d-0ad9-493d-b902-35044b4eff7d)
![example token](https://github.com/user-attachments/assets/7c9ee82a-2769-4297-9cba-600ff2d3a837)

**Comment by lumosminima** [comment]
It's all explained in the docs, give this a read: https://pocketbase.io/docs/api-rules-and-filters/ I don't understand the wording above fully but it seems like you might be looking for: https://pocketbase.io/docs/api-rules-and-filters/#examples:~:text=Allow%20only%20registered%20users%3A
